packages/create-project/tsconfig.json

```diff
@@ -2,8 +2,10 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./lib",
-    "rootDir": "./src"
+    "rootDir": "./",
```

Comment:
Does this cause generated JS files to be placed in `lib/src/`?
You can avoid this by asking typescript not to include package.json and bypassing some checks:
https://github.com/temporalio/sdk-node/blob/main/packages/worker/src/pkg.ts